### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Kutyl87**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+    GIT_REPOSITORY https://github.com/bkaradzic/bx.git
+    GIT_TAG master
+)
+FetchContent_GetProperties(bx)
+if(NOT bx_POPULATED)
+    FetchContent_Populate(bx)
+    execute_process(COMMAND git submodule update --init --recursive
+    WORKING_DIRECTORY ${bx_SOURCE_DIR})
+    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${bx_SOURCE_DIR} ${CMAKE_BINARY_DIR}/_deps/bx)
+    execute_process(COMMAND make ${PLATFORM} -C ${CMAKE_BINARY_DIR}/_deps/bx/)
```
Comment (wmlynik) [inline CMakeLists.txt]: Move compilation of library from makefile generation phase to compilation phase

Comment (Kutyl87) [reply CMakeLists.txt]: @pan-diaxik thank you, issue fixed 👍

**Review by wmlynik — COMMENTED** [review]
CMake freezes for some time while downloading bgfx, maybe add some message informing that this is normal.

**Review by wmlynik — APPROVED** [review]
Works very well

**Pull request merged**